features/base/tracks/reducer.js

```diff
@@ -7,19 +10,43 @@ import {
 } from './actionTypes';
 
 /**
- * Listen for actions that add or remove remote tracks.
+ * Listen for actions that add or remove remote and local tracks.
  */
 ReducerRegistry.register('features/base/tracks', (state = [], action) => {
     switch (action.type) {
+    /**
+     * Remove all tracks when connection is disconnected.
+     */
+    case CONNECTION_DISCONNECTED:
+        return [];
+
+    /**
+     * Remove remote tracks when conference is left.
+     */
+    case CONFERENCE_LEFT:
+        return state.filter(track => track.isLocal());
```

Comment:
I think we talked that CONFERENCE_LEFT should result in PARTICIPANT_LEFT for each & every participant. If that's what you've ensured, why would you have to remove remote tracks here if they've already been removed by PARTICIPANT_LEFT?